src/utils/get_gigs.js

```diff
@@ -12,7 +12,7 @@ const titleCase = str =>
     .join(' ');
 
 const createSubtitle = gig => {
-  const price = gig.price ? `$${gig.price}` : 'Free';
+  const price = gig.price !== 0 && gig.price !== null ? `$${gig.price}` : 'Free';
```

Comment:
Null and zero are falsy, so we don't actually need to check for them specifically.
https://developer.mozilla.org/en-US/docs/Glossary/Falsy


This will actually handle fewer cases than the line it replaces - if we have a gig with no price key it'll return `$undefined` instead of `Free`

Reply:
Interesting. Running a little test now, what you're saying makes sense & works. What prompted the change though was shows returning `$null` as the price. Eg;
<img width="306" alt="image" src="https://user-images.githubusercontent.com/15998949/129445330-2738f5dc-c071-4691-97db-64e377bc809b.png">

Noting the associated gig response is:
```json
[
    {
        "title": "The Stephen Magnusson Trio",
        "venue": "uptown_cafe",
        "description": "Unfortunately this performance is cancelled because of you know what. Stay safe all.     One of the world's leading improvised music guitar players  Guitarist and composer Stephen Magnusson is a musician at the height of his powers. From the precocious virtuoso who dazzled the music world when he crashed onto the scene, to a musician so mature, complete and respected that when people talk of an “Australian sound” of instrumental music, it is Magnusson’s influence – his tone, his touch, his compositional watermark and his deep sensitivity – that they hear.  founding member of SNAG, FESTA, Assumptions Trio (Julien Wilson and Will Guthrie) Julio and The Stevies (again with Julien and also Stephen Grant), his trio MAG (Frank DiSario and Dave Beck) and the latest quartet – Magnet (Carl Pannuzzo, Eugene Ball and Sergio Beresovsky) Steve is one of the most vital, musical and leading exponents in this music. Plus he's a sweetheart....  featuring everything from hard swinging Ornette Coleman tunes to delicate originals and distinctive ballads, sometimes even a cover of a Cold Chisel or The Police, Steve captivates and meserises with his incredible musicianship and openness....   with -   Frank Di Sario - double bass  Dave Beck – drums    2 shows - 8pm and 9.30pm   Bookings -uptownjazzcafe@email.com    More                Inside Uptown",
        "performance_date": null,
        "doors_open": "7:30 PM",
        "music_starts": "8:00 PM",
        "price": null,
        "url": "https://www.uptownjazzcafe.com",
        "image_url": "https://www.uptownjazzcafe.com/index_2145_1077310322.jpg"
    }
]
```

Reply:
That wasn't just before the price check was deployed was it? I'd only just pushed that change at about 4pm.

We could really use timestamps in our logs. I know you can decode timestamps from an objectId but I feel like it's easier to just have `createdAt` `updatedAt` properties

Reply:
Noticed the issue when the >10 bug popped up. Devved the change before frisbee at the same time as the null title fix

Reply:
Ah, yeah. This cheeky guy would've fixed it. We were only checking if it was zero prior
https://github.com/zootytooty/BeBot/commit/2fb78671f720dd99c73f6145965e6531ef8ab52d

Reply:
Ahhhh. 

Wanna push a fix?

Reply:
I got a cat in my lap. Had to use the GitHub editor on my phone.